src/lib/definition-parser.ts

```diff
@@ -143,22 +155,33 @@ export async function getTypingInfo(folderName: string): Promise<TypingParseFail
 
 interface EntryPointSuccess {
 	kind: "success";
-	filename: string;
+	filenames: string[];
 }
 interface EntryPointFailure {
 	kind: "failure";
 	message: string;
 }
 async function entryPoint(directory: string, folderName: string, log: Logger): Promise<EntryPointSuccess | EntryPointFailure> {
+	// If there is a tsconfig.json with a "files" propoerty use this as the enty point
+	if (await fsp.exists(path.join(directory, "tsconfig.json"))) {
+		const files: string[] = JSON.parse(await readFile(directory, "tsconfig.json")).files;
+		const filenames = files && files.filter(file => file.endsWith(".d.ts"));

```

Comment:
Just wrap it all in `if (files)` -- you use `filenames.length` unconditionally just below this line.